Assets/Character/Scripts/CollisionCommon.cs

```diff
@@ -0,0 +1,12 @@
+﻿using UnityEngine;
+
+public static class CollisionCommon // Better Name?

```

Comment:
okay name. May be changed depending on what other functions are added in this file